updates/32.sql

```diff
@@ -0,0 +1,18 @@
+INSERT INTO datenbank_mapping(Formular, Datenfeld, de_Beschriftung, Feldtyp, Array, Seite, gb_beschriftung, fr_beschriftung, la_beschriftung)  VALUES ("freie_suche", "Seite", "Seite", "textfield", 0, "freie_suche", "page", "page", "pagina");
+
+UPDATE datenbank_mapping SET de_Tooltip = 'Die NeG-ID besteht aus einem Buchstaben für die Art des Datensatzes (B=Einzelbeleg, E=Edition, M=MGH-Lemma, N=NamenLemma, P=Person, Q=Quelle) + laufender Nummer (z. b. P 7404 = Karl der Große). Geben Sie hier eine vollständige ID ein. Die ID finden Sie oben rechts in jedem Datensatz-Eintrag' WHERE ID = 276;
+
+ALTER TABLE ueberlieferung_edition
```

Comment:
The name of the foreign key should stay the same. If this is not possible within one ALTER TABLE statement, please split it up into 2 statements.

However i am not entirely sure whether we should add these constraints right now because we decided to discuss it in detail with Mr. Fischer before we introduce any ON DELETE CASCASE operations. Maybe we can discuss this together on tuesday with him.

Reply:
erledigt